README.md

```diff
@@ -7,21 +7,131 @@
 > A simple Cryptography Implementation in Python for the ARK Blockchain.
 
 [![Build Status](https://img.shields.io/travis/ArkEcosystem/python-crypto/master.svg?style=flat-square)](https://travis-ci.org/ArkEcosystem/python-crypto)
+[![Codecov](https://img.shields.io/codecov/c/github/arkecosystem/python-crypto.svg)](https://codecov.io/gh/arkecosystem/python-crypto)
 [![Latest Version](https://img.shields.io/github/release/ArkEcosystem/python-crypto.svg?style=flat-square)](https://github.com/ArkEcosystem/python-crypto/releases)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
+# Contributions are closed
+
+We will not be accepting new PRs until we are happy with the base of the client and until it has good test coverage. We'll let you know on our #python Slack channel once we'll accept PRs again.
+
+### AIP11 Serialization
+- [x] Transfer
+- [x] Second Signature Registration
+- [x] Delegate Registration
+- [x] Vote
+- [x] Multi Signature Registration
+- [x] IPFS
+- [x] Timelock Transfer
+- [x] Multi Payment
+- [ ] Delegate Resignation
+
+### AIP11 Deserialization
+- [x] Transfer
+- [x] Second Signature Registration
+- [x] Delegate Registration
+- [x] Vote
+- [x] Multi Signature Registration
+- [ ] IPFS
+- [ ] Timelock Transfer
+- [ ] Multi Payment
+- [ ] Delegate Resignation
+
+### Transaction Signing
+- [x] Transfer
+- [x] Second Signature Registration
+- [x] Delegate Registration
+- [x] Vote
+- [x] Multi Signature Registration
+
+### Transaction Verifying
+- [x] Transfer
+- [x] Second Signature Registration
+- [x] Delegate Registration
+- [x] Vote
+- [x] Multi Signature Registration
+
+### Transaction Entity
+- [x] getId
+- [x] sign
+- [x] secondSign
+- [x] parseSignatures
+- [x] serialize
+- [x] deserialize
+- [x] toBytes
+- [x] toArray - to_dict in Python
+- [x] toJson
+
+### Message
+- [x] sign
+- [x] verify
+- [x] toArray - to_dict in Python
+- [x] toJson - it returns a dictionary object
+
+### Address Identity
+- [x] fromPassphrase
+- [x] fromPublicKey
+- [x] fromPrivateKey
+- [x] validate
+
+### Private Key Identity
+- [x] fromPassphrase
+- [x] fromHex
+
+### Public Key Identity
+- [x] fromPassphrase
+- [x] fromHex
+
+### WIF Identity
+- [x] fromPassphrase
+
+### Configuration
+- [x] getNetwork
+- [x] setNetwork
+- [x] getFee
+- [x] setFee
+
+### Slot
+- [x] time
+- [x] epoch
+
+### Networks (Mainnet, Devnet & Testnet)
+- [x] epoch
+- [x] version
+- [x] nethash
+- [x] wif
+
 ## Installation
 
 ```bash
 ...
 ```
 
+## Guide for contributing
+
+Before you start contributing please take some time and check our official [Python Development Guidelines](https://github.com/ArkEcosystem/development-guidelines/blob/master/Python/README.md) which follow the widely accepted PEP8 Python Style Guide. 🖋
```

Comment:
We should move this to docs.ark.io in a section with rough guidelines for every language @mvega3